crates/bevy_reflect/src/enums/dynamic_enum.rs

```diff
@@ -140,6 +140,19 @@ impl DynamicEnum {
         self.variant = variant.into();
     }
 
+    pub fn variant(&self) -> &DynamicVariant {
+        &self.variant
+    }
+
+    /// Using the mut reference to switch to a different variant will ___not___ update the
```

Comment:
Nit: doc comments typically start with a one sentence summary of what the method does, followed by an empty line, and then these sorts of details. Would you be able to update this to follow that convention here?